### PR TITLE
ipam: add Contains

### PR DESCRIPTION
**Pull request by kopiczko**
This is useful for making assertions in validating custom objects.

### CODE REVIEW

File: ipam_test.go [hunk 1]
```diff
@@ -534,6 +534,72 @@ func TestSpace(t *testing.T) {
 	}
 }
 
+func TestConains(t *testing.T) {
+	tests := []struct {
+		Network          string
+		Subnet           string
+		ExpectedContains bool
+	}{
+		{ // Test 0.
```
Comment (xh3b4sd) [inline ipam_test.go]: I would like comments to have their own line.

Comment (kopiczko) [reply ipam_test.go]: Done.

File: ipam.go [hunk 2]
```diff
@@ -126,6 +126,13 @@ func space(freeIPRanges []ipRange, mask net.IPMask) (net.IP, error) {
 	return nil, microerror.Maskf(spaceExhaustedError, "tried to fit: %v", mask)
 }
 
+// Contains returns true when the subnet is a part of the network, false
+// otherwise.
+func Conains(network, subnet net.IPNet) bool {
```
Comment (JosephSalisbury) [inline ipam.go]: `Contains` plz

**Comment by kopiczko** [comment]
Tested on godsmack.

**Comment by kopiczko** [comment]
We don't need that anymore. azure-operator computes subnetes on its own. This is way better.

**Pull request closed without merging**